src/components/Dashboard/TodoList.vue

```diff
@@ -2442,6 +2523,8 @@ export default class TodoList extends Mixins(AllowableActionsMixin, DateMixin) {
 
 textarea {
   color: $gray7;
+  background-color: $gray1;
+  padding: 10px 10px 10px 20px;
```

Comment:
If it works, I recommend moving this padding to the textarea component in the HTML and using the Vuetify helper classes.

By the way, I think should specify 20px right padding, too. (See px and py classes.)

Reply:
Updated